.github/workflows/dailyWorkflow.yml

```diff
@@ -47,36 +47,21 @@ jobs:
           uses: ruby/setup-ruby@v1
           with:
             ruby-version: '3.2'
-        - name: Install dependencies
+        - name: Generate report
```

Comment:
Which report? Maybe something along the lines of `weekday gh issue report`?

Reply:
I changed the names of each step so it gives extra details of what each of them do